examples/angular/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "angular",
+  "version": "0.0.0",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "watch": "ng build --watch --configuration development",
+    "test": "ng test"
+  },
+  "private": true,
+  "dependencies": {
+    "@angular/animations": "~13.2.0",
+    "@angular/common": "~13.2.0",
+    "@angular/compiler": "~13.2.0",
+    "@angular/core": "~13.2.0",
+    "@angular/forms": "~13.2.0",
+    "@angular/platform-browser": "~13.2.0",
+    "@angular/platform-browser-dynamic": "~13.2.0",
+    "@angular/router": "~13.2.0",
+    "near-wallet-selector": "*",
+    "process": "^0.11.10",
```

Comment:
Little concerned about using this module as it hasn't changed in 5 years! Are we able to simply polyfill process to an empty object or maybe `{ env: {} }`?

Reply:
I tried to do it this way but for some reason I could not get it working and when I tried to send messages without a "donation" `near-api-js` was throwing a strange error which in the end I found it was looking for `process.env.SOME_VARIABLE` and I lost a few hours thinking env variables were working properly :(